packages/get-packages/src/index.ts

```diff
@@ -6,21 +6,19 @@ import fs from "fs-extra";
 import path from "path";
 import globby, { sync as globbySync } from "globby";
 import readYamlFile, { sync as readYamlFileSync } from "read-yaml-file";
-import { PackageJSON } from "@changesets/types";
 import { findRoot, findRootSync } from "@manypkg/find-root";
-
-const isNoEntryError = (err: unknown): boolean =>
-  !!err && typeof err === "object" && "code" in err && err.code === "ENOENT";
-
-export type Tool = "yarn" | "bolt" | "pnpm" | "lerna" | "root";
```

Comment:
note that `'root'` -> `'singlePackage'` can potentially be a breaking change if `.type` is returned anyhow to the caller

Reply:
This is going to be a breaking change anyway because packages that used manypkg before will need to adjust a check like `monorepo.tool === "root"` to `monorepo.tool.type === "root"` -- but I agree that changing the property name (and not the values) does make this a much easier transition.

I've renamed the tool back to "root" / "RootTool", to ease initial transition to the new structure.

Reply:
Ah, ye - good point about the changed structure. It's not like we have a lot of API consumers though, so I don't think that we have to be too cautious when it comes to breaking changes. If we believe that the new name was better, let's keep it. It's better to clean up what we can now, in a single sweep - rather than to drag it through multiple major versions or keep the confusing name forever.